rest-api/dao/participant_summary_dao.py

```diff
@@ -73,18 +73,33 @@
 
 _PARTICIPANT_ID_FILTER = " WHERE participant_id = :participant_id"
 
+_WHERE_SQL = """
+not sample_status_%(test)s_time <=>
```

Comment:
Note: we'll want to test this in test manually after merging, since we're using something MySQL-specific with <=>.